components/core/DpDataTable/DpTableHeader.vue

```diff
@@ -1,5 +1,14 @@
+<license>
+  (c) 2010-present DEMOS E-Partizipation GmbH.
+
+  This file is part of the package @demos-europe/demosplan-ui,
+  for more information see the license file.
+
+  All rights reserved
+</license>
+
```

Comment:
Didn't we want to drop the license-header?

Reply:
why should we?

Reply:
It's not necessary to have it in there for MIT compliance and it has been known to cause issues with build tooling. That's why we decided to drop it.

Reply:
But this is demosplan and not MIT, thus the header has to stay.

Reply:
Ok. This needs to be documented somewhere prominently that we assessed this and dropped it. Is this already the case?

Reply:
removed this leftover license tag in [9754f51](https://github.com/demos-europe/demosplan-ui/pull/13/commits/9754f51810b3dfa98093e5d8e5f7243f4d11ed87)